### PR TITLE
fixtures: generate new files

### PR DESCRIPTION
**Pull request by BadrAly**
* Replaces big 10k (items, holdings, docs) files by small 1k files.
* Improves code to insert pids to identifier table.
* Adds extra language codes, needed for the transformation.
* Fixes import problem for the transformation.

Co-Authored-by: Aly Badr aly.badr@rero.ch

### CODE REVIEW

File: rero_ils/dojson/cli.py [hunk 1]
```diff
@@ -17,6 +17,7 @@
 
 """Click command-line utilities."""
 
+import json
```
Comment (jma) [inline rero_ils/dojson/cli.py]: why?

File: rero_ils/modules/documents/jsonschemas/documents/document-minimal-v0.0.1.json [hunk 2]
```diff
@@ -89,18 +89,21 @@
             "description": "Language value.",
             "type": "string",
             "enum": [
-              "fre",
```
Comment (jma) [inline rero_ils/modules/documents/jsonschemas/documents/document-minimal-v0.0.1.json]: why did you change the list of languages? If it is really the case, you need to change also the form option.

File: rero_ils/dojson/cli.py (hunk 1 above)
Comment (BadrAly) [reply rero_ils/dojson/cli.py]: This is to correct a bug when generating the json file.

File: rero_ils/modules/documents/jsonschemas/documents/document-minimal-v0.0.1.json (hunk 2 above)
Comment (BadrAly) [reply rero_ils/modules/documents/jsonschemas/documents/document-minimal-v0.0.1.json]: As discussed, adding a note to the commit message

**Comment by BadrAly** [comment]
> It can be a good idea to change the fixture file names to something more explicit such as big instead of 1k, small instead of 300. Includes the resource name in the fixtures file name would be also good.

done

File: scripts/setup [hunk 3]
```diff
@@ -115,12 +118,12 @@ pipenv run invenio fixtures create --pid_type doc --schema 'http://ils.rero.ch/s
 
 # create items
 # NEEDS item type !!!
-# to generate 10k items file 10k documents must exist in DB
-# pipenv run invenio fixtures create_items -i 5 -t ${DATA_PATH}/items10k.json -h ${DATA_PATH}/holdings10k.json
-# pipenv run flask utils check_json -r data/holdings10k.json
+# to generate 1k items file 1k documents must exist in DB
```
Comment (rerowep) [inline scripts/setup]: 1change to `1k` and `big`

File: scripts/setup [hunk 4]
```diff
@@ -115,12 +118,12 @@ pipenv run invenio fixtures create --pid_type doc --schema 'http://ils.rero.ch/s
...
+# pipenv run invenio fixtures create_items -i 2 -t ${DATA_PATH}/items_big.json -h ${DATA_PATH}/holdings_big.json
+# pipenv run flask utils check_json -r data/holdings_big.json
 # to generate 300 items file 300 documents must exist in DB
```
Comment (rerowep) [inline scripts/setup]: change `300` to `small`
